content/documentation/pages/1-installation/1-local/3-manual.md

```diff
@@ -110,13 +110,15 @@ All the prebuilt streaming applications:
 Applications can be registered individually using the `app register` command or in bulk using the `app import` command.
 There are also bulk-registration links that represent the group of prebuilt applications for a specific release which is useful for getting started.
 
-You can register applications using the UI or the shell.
+You can register stream and task applications using the UI or the shell.
 
-Depending on if you are using RabbitMQ or Kafka, register the applications using the respective URLs.
+For streams, depending on if you are using RabbitMQ or Kafka, register the applications using the respective URLs.
```

Comment:
Maybe change the order of rabbit vs. kafka, so the links beneath are consistent with the order.